features/desktop/report.feature

```diff
@@ -0,0 +1,44 @@
+@javascript
+Feature: reporting of posts and comments
+
+  Background:
+    Given a user with username "bob"
+    And a moderator with email "alice@alice.alice"
+    And "bob@bob.bob" has a public post with text "I'm a post by Bob"
+    And the terms of use are enabled
+
+  Scenario: User can report a post, but cannot report it twice
+    Given I sign in as "alice@alice.alice"
+    And I am on the public stream page
+    When I hover over the ".stream-element"
+    And I click to report the post
+    Then I should see the report modal
+    And I should see "You should only report posts or comments which violates"
+    When I fill in "report-reason-field" with "That's my reason"
+    And I submit the form
+    Then I should see a success flash message containing "The report has successfully been created"
+    When I hover over the ".stream-element"
+    And I click to report the post
+    And I fill in "report-reason-field" with "That's my reason2"
+    And I submit the form
+    Then I should see an error flash message containing "The report already exists"
+    When I go to the report page
```

Comment:
I still don't succeed to go to that page. I have the error message:
`No route matches {:action=>"update", :controller=>"report"}, missing required keys: [:id] (ActionController::UrlGenerationError)`

I first thought it was because terms weren't enabled but I enable them in the Background section, and it doesn't change anything. I need help here.

Reply:
up on this, @SuperTux88 could you please explain to me:
- why there is no get route defined in routes.rb, but accessing /report currently works
- why capybara is trying to navigate to /report with the "update" action instead of doing a simple get?